Public/Src/Engine/Processes/SandboxedProcessMac.cs

```diff
@@ -396,7 +396,7 @@ private async Task FeedStdInAsync(SandboxedProcessInfo info, [CanBeNull] string
 
             string line = I($"exec {info.FileName} {escapedArguments} {redirectedStdin}");
 
-            LogProcessState("Feeding stdin: " + line);
+            LogProcessState("Feeding stdin");

```

Comment:
>LogProcessState("Feeding stdin"); [](start = 12, length = 33)

Is this useful at this point?

Reply:
not really.  It takes a lot of space in the text log.  It's available in the XLG when needed.